src/content/docs/nrql/nrql-syntax-clauses-functions.mdx

```diff
@@ -1521,26 +1521,112 @@ SELECT histogram(duration, 10, 20) FROM PageView SINCE 1 week ago
   <Collapser
     className="freq-link"
     id="func-histogram"
-    title={<InlineCode>histogram(attribute, ceiling [,number of buckets])</InlineCode>}
-  >
-    Use the `histogram()` function to generate histograms. It takes three arguments:
-
-    * Attribute name
-    * Maximum value of the sample range
-    * Total number of buckets (between 1 and 500, inclusive)
+    title={<InlineCode>histogram(attribute[, width: n][, buckets: n][, start: n])</InlineCode>}
+  >
+    Use  the `histogram()` function to generate histograms, which are useful for
+    visualizing the distribution of a dataset. It divides the dataset into a
+    specified number of buckets and counts the number of data points that fall
+    into each bucket.
+
+    #### Arguments:
+    * `attribute` The first argument is required and specifies the attribute for
+      which to count values that fall within each histogram bucket range.
+    * `width:` Inidicates the width of the sample range. The maximum value of the range is the `start` argument value plus this `width` value.
+      * When using positional (non-labeled) arguments, `width` is the second argument.
+      * _Default:_ 10
+    * `buckets:` Total number of buckets (between 1 and 500, inclusive).
+      * When using positional (non-labeled) arguments, `buckets` is the third argument.
+      * _Default:_ 40
+    * `start:` The beginning of the histogram range.
+      * When using positional (non-labeled) arguments, `start` is the fourth argument.
+      * _Default:_ 0
+
+
+    #### First and last buckets include out of range values
+    Values that fall out side the defined histogram range are included in the
+    first or last buckets. The first bucket count will include items that
+    precede the histogram range, and the last bucket count will include items
+    that succeed the histogram range. To exclude these values from the histogram results,
```

Comment:
It would probably be clearer to say "are smaller than" and "are larger than" rather than precede & succeed.